MdeModulePkg/MdeModulePkg.dec

```diff
@@ -1272,6 +1272,14 @@
   gEfiMdeModulePkgTokenSpaceGuid.PcdSupportAlternativeQueueSize|FALSE|BOOLEAN|0x40000151
   # MU_CHANGE [END]
 
+  # MU_CHANGE [BEGIN] - Support infinite boot retries
+  # #  If enabled, Bds will retry boot options indefinitely

```

Comment:
I think this PCD needs more documentation. Mainly explaining why this PCD exists. MdeModulePkg is one of the most common package dependencies so many people will see this so I think it should be written to precisely explain why the PCD exists and when it should be used.

Reply:
Updated documentation to try to better describe the usage and intention.